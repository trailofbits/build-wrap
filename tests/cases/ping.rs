use std::process::{exit, Command, ExitStatus};

fn main() {
    let status: ExitStatus = Command::new("ping")
        .args(["-c", "1", "-v", "127.0.0.1"])
        .status()
        .unwrap();
    let code = status.code().unwrap();
    exit(code);
}
