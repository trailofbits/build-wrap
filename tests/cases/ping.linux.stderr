ping: socket: Operation not permitted
